README.md

```diff
@@ -135,42 +135,72 @@ In order to see the command line options, you can use:
 
 ```bash
 /path/to/isntall/directory/coreneuron_exec --help
+-b, --spikebuf ARG       Spike buffer size. (100000)
+-c, --threading          Parallel threads. The default is serial threads.
+-d, --datpath ARG        Path containing CoreNeuron data files. (.)
+-dt, --dt ARG            Fixed time step. The default value is set by
+                         defaults.dat or is 0.025.
+-e, --tstop ARG          Stop time (ms). (100)
+-f, --filesdat ARG       Name for the distribution file. (files.dat)
+-g, --prcellgid ARG      Output prcellstate information for the gid NUMBER.
+-gpu, --gpu              Enable use of GPUs. The default implies cpu only run.
+-h, --help               Print a usage message briefly summarizing these
+                         command-line options, then exit.
+-i, --dt_io ARG          Dt of I/O. (0.1)
+-k, --forwardskip ARG    Forwardskip to TIME
+-l, --celsius ARG        Temperature in degC. The default value is set in
+                         defaults.dat or else is 34.0.
+-mpi                     Enable MPI. In order to initialize MPI environment this
+                         argument must be specified.
+-o, --outpath ARG        Path to place output data files. (.)
+-p, --pattern ARG        Apply patternstim using the specified spike file.
+-R, --cell-permute ARG   Cell permutation, 0 No; 1 optimise node adjacency; 2
```

Comment:
cell_permute 0 and 1 work on cpu and gpu Is useful for performance comparison and testing.
Cell permute 2 works only for gpu.  On gpu 1 is much faster than 0 and 2 is best. So
a default of 1 is reasonable.